public/app.js

```diff
@@ -11,7 +11,7 @@ import route from 'can/route/';
 import Session from './models/session';
 import "can/route/pushstate/";
 import stache from "can/view/stache/";
-import "./util/prefilter";
```

Comment:
why would this change?  Shouldn't there a be a prefilter.js alongside our converter's module?